lib/migrate/Migrator.js

```diff
@@ -475,7 +474,7 @@ class Migrator {
   _waterfallBatch(batchNo, migrations, direction, trx) {
     const trxOrKnex = trx || this.knex;
     const { tableName, schemaName, disableTransactions } = this.config;
-    let current = Bluebird.bind({ failed: false, failedOn: 0 });
+    let current = Promise.resolve();
```

Comment:
are those fields not used anywhere?

Reply:
I didn't found any usages in project

Reply:
7a6937fdb329d0d0b29cd51fc28aea06e6903e0e first revision where failedOn exists, i think it just partially implemented feature

Reply:
Fair enough, guess we can drop it.